src/main/java/frc/robot/Robot.java

```diff
@@ -75,6 +75,7 @@ public void robotPeriodic() {
    */
   @Override
   public void disabledInit() {
+    Blinkin.blinkTeamColors();
```

Comment:
Technically the blinkin can't actually blink in disabled because it's technically a motor and you can't control the motor in disabled.